app/retail/emails.py

```diff
@@ -1021,14 +1021,87 @@ def render_new_bounty_roundup(to_email):
pVfjj9Ri56AvRT98H6EB4t0epr6FJWLMUvP-2FYoUHNMACW7qGgCAAkW4-2FLiQO4hUuRoKbRLMw5uap9fviiF1iKCsooI23yTUqsWigzoK-2BWMkCNZAZDD5gzbZ2-2BSc5UZFLAtnyahkJqxzu2PAWz3shYV0-2FvrlpwtvhI5emrVqca0tUc8x&amp;source=gmail&amp;ust=1586984596249000&amp;usg=AFQjCNGi1ILCw6ktN0tYTLP_MYhdBNzv4Q"><img src="https://ci5.googleusercontent.com/proxy/BSTArez13C2mcKUDmLQGaJyiC1pibLCvQHJNAVbnP8lnae7iN-e1lH7SstdZewWKidXQEK2zLgJN_w=s0-d-e1-ft#https://gitcoin.co/dynamic/kudos/11602/" class="CToWUd"></a><a href="https://u5829119.ct.sendgrid.net/ls/click?upn=JzKZkqL7BQlM7N5Fr31jcGuw591LiVuDWNcwacQCseOb5oHHByPrFZJRwluGnvd46djX_5zLnBF78PFiWL6TQQilAz27oyL2C0F-2FIQb4jxFPSdKhphqySxhcrrX2kp5efAfttfl0xD2fU65of0HE9amY4DGQwCrjpdzlng9XEDT-2BRaCTRWOXMecRJWQUKtCWA8jidrCJJQvtT20KInZsf-2FnvXjk5i-2FW9J-2BuSATrPfAD3EG6djmu1F9Sd00MUaO2qfrYPp2mzB-2BkQaj0KYEYxj1zK-2BlhLlcPLxO318iHpK6evwhZcxUFTzBYJQeve0dtdQ9laE2dkwnos5KiHBIYhC83KaY6jZJRkNYSqQJ3G66soTRPtdj2Rl91C63IxwBJP4QdDc" target="_blank" data-saferedirecturl="https://www.google.com/url?q=https://u5829119.ct.sendgrid.net/ls/click?upn%3DJzKZkqL7BQlM7N5Fr31jcGuw591LiVuDWNcwacQCseOb5oHHByPrFZJRwluGnvd46djX_5zLnBF78PFiWL6TQQilAz27oyL2C0F-2FIQb4jxFPSdKhphqySxhcrrX2kp5efAfttfl0xD2fU65of0HE9amY4DGQwCrjpdzlng9XEDT-2BRaCTRWOXMecRJWQUKtCWA8jidrCJJQvtT20KInZsf-2FnvXjk5i-2FW9J-2BuSATrPfAD3EG6djmu1F9Sd00MUaO2qfrYPp2mzB-2BkQaj0KYEYxj1zK-2BlhLlcPLxO318iHpK6evwhZcxUFTzBYJQeve0dtdQ9laE2dkwnos5KiHBIYhC83KaY6jZJRkNYSqQJ3G66soTRPtdj2Rl91C63IxwBJP4QdDc&amp;source=gmail&amp;ust=1586984596249000&amp;usg=AFQjCNG0veChh_xAg5KKCXw-5ERrb-1xog"><img src="https://ci4.googleusercontent.com/proxy/dSvCLCzQzKLWfVImFZFwgChOOoQ38m6DuukVu_OPBSTPdhUkLRPxL8MEWe2qDeoLxkLgjrtF8dHX=s0-d-e1-ft#https://gitcoin.co/dynamic/kudos/7496/" class="CToWUd"></a><a href="https://u5829119.ct.sendgrid.net/ls/click?upn=JzKZkqL7BQlM7N5Fr31jcGuw591LiVuDWNcwacQCseMwhgfkTLghKGS-2BI8dJQVbqDVpQ_5zLnBF78PFiWL6TQQilAz27oyL2C0F-2FIQb4jxFPSdKhphqySxhcrrX2kp5efAfttfl0xD2fU65of0HE9amY4DAyTOt7DQ-2BZfgiN3Y6ugsZvNaNjMRMnH9xWv-2B6oSfwhcyw5ahwYcAqf2-2BZNfGK1nOdKfVHmAVMQ52-2FSPMCrisGSdYyhkRf7xdqBsYBrebXrqi-2FUmj2gwjZvpi8EZOZjZSlZqmEuacwGlmcWNSzFHTt5sbPmlMsCAwv2O806ILZA3trqtt8Jgd2aFie1VZsKRZNoQe6WYq5oh5z0LmE88xm92H1uWbmdFDXmhqlY8Ma-2Fo" target="_blank" data-saferedirecturl="https://www.google.com/url?q=https://u5829119.ct.sendgrid.net/ls/click?upn%3DJzKZkqL7BQlM7N5Fr31jcGuw591LiVuDWNcwacQCseMwhgfkTLghKGS-2BI8dJQVbqDVpQ_5zLnBF78PFiWL6TQQilAz27oyL2C0F-2FIQb4jxFPSdKhphqySxhcrrX2kp5efAfttfl0xD2fU65of0HE9amY4DAyTOt7DQ-2BZfgiN3Y6ugsZvNaNjMRMnH9xWv-2B6oSfwhcyw5ahwYcAqf2-2BZNfGK1nOdKfVHmAVMQ52-2FSPMCrisGSdYyhkRf7xdqBsYBrebXrqi-2FUmj2gwjZvpi8EZOZjZSlZqmEuacwGlmcWNSzFHTt5sbPmlMsCAwv2O806ILZA3trqtt8Jgd2aFie1VZsKRZNoQe6WYq5oh5z0LmE88xm92H1uWbmdFDXmhqlY8Ma-2Fo&amp;source=gmail&amp;ust=1586984596249000&amp;usg=AFQjCNFh4DGlmacZcRPxwyJvM6IGHZNtpA"><img src="https://ci3.googleusercontent.com/proxy/ovCZsoJjeRuwhwzwpmSMyXWoDb3CdXyaWsxJP-KbgwlO4jWnbmHeUN7rd23H3sol-9jlEtc_6DTv=s0-d-e1-ft#https://gitcoin.co/dynamic/kudos/1838/" class="CToWUd"></a>
```

Comment:
whats happening with these links?

Reply:
Those are just dummy data to place the kudos images. I shall replace them.